src/_drivers.ts

```diff
@@ -140,3 +142,24 @@ export const builtinDrivers = {
   "vercel-runtime-cache": "unstorage/drivers/vercel-runtime-cache",
   "vercelRuntimeCache": "unstorage/drivers/vercel-runtime-cache",
 } as const;
+
+export type BuiltinDrivers = {
+  "deno-kv": DenoKVDriver;
+  "denoKV": DenoKVDriver;
+}
+
+export type DriverGetOptions = {
+  "deno-kv"?: DenoKVDriver extends { getOptions: infer TGet } ? unknown extends TGet ? {} : TGet : {}
```

Comment:
This is not easily doable in runtime to pass `{ 'deno-kv': { ... opts } }`, we should instead use `{ denoKV: {} }` to be easier with JS objects.

Reply:
PS: We could simplify the interface to the drivers to actually accept a custom option. We can start by denoKV as an example, but then we need to allow actually `tOpts.denoKV.*`.

Like discussed, inference of name could be in followup PR. This is mainly to document a "bag of possibe options" for get/set/etc